sample/ApplicationInsightsHostingStartupSample/Views/Home/ScriptCheck.cshtml

```diff
@@ -0,0 +1,108 @@
+﻿@{
+    ViewData["Title"] = "Home Page";
+}
+
+<div id="myCarousel" class="carousel slide" data-ride="carousel" data-interval="6000">
+    <ol class="carousel-indicators">
```

Comment:
@pakrym I think it's fine given that that's the most realistic scenario for this. I agree it's not technically needed.